mbuild/recipes/polymer.py

```diff
@@ -68,6 +73,16 @@ def __init__(self, monomers, n, sequence='A', port_labels=('up', 'down')):
         # Hoist the first part's bottom port to be the bottom port of the polymer.
         self.add(first_part.labels[port_labels[1]], port_labels[1], containment=False)
 
+        # Add a cap to left/right or both ends of the pattern i.e. C->ABABAB
+        for cap, c_port, end, e_port in zip(caps,cap_ports,
+        [first_part,last_part],[1,0]):
```

Comment:
You shouldn't need to loop over `[first_part, last_part]` here. Since in [the lines above](https://github.com/Gawquon/mbuild/blob/491e773579fdb7cf8a80598c8de5856949803266/mbuild/recipes/polymer.py#L70-L74) we are adding the port labels at the two ends to `Polymer`.

I think instead you can just do:
```
for cap, cap_port, end_port in zip(
        caps,
        cap_ports,
        reversed(port_labels)):
```